doc/overview/build.md

```diff
@@ -0,0 +1,127 @@
+# Endless OS 7 build process
+
+This document gives an overview of the tools used to build Endless OS 7.
+
+For guidance on how to build EOS7, see: [`doc/howto/build.md`](doc/howto/build.md).
+
+## OSTree build stage
+
+This stage is defined in
+[eos-build-meta.git](https://github.com/endlessm/eos-build-meta/) using the BuildStream build tool.
+
+It uses some 3rd party BuildStream plugins, which are listed in `project.conf`.
+
+It uses elements from Freedesktop SDK and GNOME Build Metadata via the junction
+elements `freedesktop-sdk.bst` and `gnome-build-meta.bst`.
+
+More details on each of these concepts below.
+
+### BuildStream
+
+BuildStream's documentation is available at:
+<https://docs.buildstream.build/>.
+
+Note that BuildStream is a plugin-based tool. Source fetching is done via
+*source plugins*, and element builds are handled by *element plugins*.
+
+The core of BuildStream ships a small set of plugins. Their documentation
```

Comment:
Ship *with*? /gen I'm not trying to teach you your language, I've never seen this used that way. :slightly_smiling_face:

Reply:
I suppose i'm implying "the core of BuildStream" is able to actually ship software, which is not true, but grammatically it's ok.